docs/tutorials/exporting_measurements.md

```diff
@@ -45,14 +45,14 @@ If your analysis involves combining measurements from different images, it is re
 
 The cleanest way to export different types of measurements in QuPath across multiple images is with the **Measurement Exporter**.
 
-Provided that your images are stored in a {doc}`project <../tutorials/projects>`, you can access it through {menuselection}`Measure --> Export measurements` (as well as {{ icon_table }} --> Export measurements).
+Provided that your images are stored in a {doc}`project <../tutorials/projects>`, you can access it through {menuselection}`Measure --> Export measurements`.
 
 :::{figure} images/measurement_exporter.png
```

Comment:
This doesn't display any more because the updated dialog is a .jpg rather than .png.

Since there are so many images, I try to use

* PNG for dialogs and graphics with lots of flat colors (since they compress well, and look sharper)
* JPEG for screenshots that show an image with more variation (so a PNG would be much larger)

So I'd change the screenshot to a PNG

Reply:
Ah so that's the reason there was a mix! Thanks